src/main/java/seedu/address/logic/commands/student/StudentDeleteCommand.java

```diff
@@ -20,36 +22,44 @@ public class StudentDeleteCommand extends Command {
     public static final String COMMAND_WORD = "studentDelete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the student identified by the index number used in the displayed person list.\n"
+            + ": Deletes the student identified by the index number (separated by whitespace)"
+            + "used in the displayed person list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
```

Comment:
```suggestion
            + "Example: " + COMMAND_WORD + " 1 2 4";
```

I think this would better illustrate how to specify multiple indices.